qa/L0_storage_S3/test.sh

```diff
@@ -416,6 +417,41 @@ aws configure set default.region $AWS_DEFAULT_REGION && \
     aws configure set aws_access_key_id $AWS_ACCESS_KEY_ID && \
     aws configure set aws_secret_access_key $AWS_SECRET_ACCESS_KEY
 
+# Clean up bucket contents
+aws s3 rm "${BUCKET_URL_SLASH}" --recursive --include "*"
+
+# Test case where S3 folder has >1000 files
+rm -rf models
+
+mkdir -p models/model/1
+echo $'class TritonPythonModel:\n\tdef execute(self):\n\t\tpass' > ./models/model/1/model.py
+
+for i in {1..1000}; do
```

Comment:
Can you set the end to be a bit larger so it is obvious that the model file will be in "2nd" page?

Reply:
Sure thing, done!